app/uk/gov/hmrc/customs/notification/receiver/controllers/CustomsNotificationReceiverController.scala

```diff
@@ -150,7 +150,7 @@ class CustomsNotificationReceiverController @Inject()(logger : CdsLogger,
 
     val result = statusCode match {
       case code if (code >= 300) && (code < 400) =>
-        Redirect(routes.CustomsNotificationReceiverController.post(), code)
+        Redirect("/customs-notifications-receiver-stub/pushnotifications", code)
```

Comment:
is the routes way no longer correct? is hardcoding the way now?

Reply:
for whatever reason it doesn't work anymore. I'm pretty sure the replacement does the same thing. Not exactly sure what changed to break the original

Reply:
routes.CustomsNotificationReceiverController.post() works for me.  Perhaps a building issue in your IDE.

Reply:
interesting, reverted 👍